src/components/data/questionsData.json

```diff
@@ -1,94 +1,278 @@
+  },
+  {
+    "id": 2,
+    "question": { 
+      "text": "Single Family Zoning is being argued in the Supreme Court. Luckily you know a few of the Justices and they will listen to your opinion. What do you want to tell them?",
       "image": "q2_trees_2.png",
       "options": [{
-        "choiceid": 3,
-        "option": "Choose and move to 4",
-        "nextId": 2
-      },
-      {
         "choiceid": 4,
-        "option": "Choose and move to 5",
-        "nextId": 3
-      }
-    ]
+        "option": "The reasons people live in different types of housing is economic, this zoning segregates people and exclusionary zoning should be struck down.",
+        "densityEffect": 1,
+        "infrastractureEffect": 0
+        },
+        {
+          "choiceid": 5,
+          "option": "It's important to protect the character of a neighborhood. If a majority of people in a region want to exclude certain housing types, they should be free to do so.",
+          "densityEffect": -1,
+          "infrastractureEffect": 0
+        },
+        {
+          "choiceid": 6,
+          "option": "There's more money to be made in a neighborhood if different housing types are available.  Exclusionary zoning isn't economically smart, strike it down and deem it unconstitutional.",
+          "densityEffect": -1,
+          "infrastractureEffect": 0
+        },
+        {
+          "choiceid": 7,
+          "option": "This isn't happening in my town so I don't think it's right for me to have an opinion here.",
+          "densityEffect": -1,
+          "infrastractureEffect": 0
+        }
+      ]
     },
-    {
-      "id": 3,
-      "question": "What do you choose?",
+    "event": "The town university is so popular that it wants to expand.",
+    "fact": "In 1926, the Supreme Court upheld exclusionary zoning in a vote of 6-3 during the case Euclid v. Ambler. Exclusionary zoning today is also known as Euclidean zoning for this reason."
+  },
+  {
+    "id": 3,
+    "question":{
+      "text": "Your coworker bought a plot of land in your neighborhood. They want to split it into two and build four tiny homes - one to live in and 3 to rent out. They are complaining to you that the city won't let them because there is a minimum lot size that a house can be in the region. He asked if you could influence city council to lower the size to allow for a tiny home. What do you want to do?",
       "image": "q3_earthquake_1.png",
       "options": [{
-        "choiceid": 5,
-        "option": "Choose and move to 2",
-        "nextId": 2
-      },
-      {
-        "choiceid": 6,
-        "option": "Choose and move to 3",
-        "nextId": 3
-      }
-    ]
+        "choiceid": 8,
```

Comment:
choice ids should start at 1 for every question :) also please capitalize the i in `id` for all of these keys 😆

Reply:
roger that